Homework4/ParseTree/ParseTree.Tests/ParseTreeTest.cs

```diff
@@ -0,0 +1,47 @@
+namespace ParseTree.Tests;
+
+using NUnit.Framework;
+using System;
+using ParseTree;
+
+
+public class Tests
```

Comment:
```suggestion
using ParseTree;

public class Tests
```
Две пустые строки подряд нехорошо